dotCMS/src/main/java/com/dotcms/browser/BrowserAPI.java

```diff
@@ -0,0 +1,721 @@
+    	return getFolderContent(usr, folderId, offset, maxResults, filter, mimeTypes, extensions, true, showArchived, noFolders, onlyFiles, sortBy, sortByDesc, languageId);
+    }
+
+    /**
+     * @deprecated see {@link #getFolderContent(BrowserQuery)}
+     * @param user
+     * @param folderId
+     * @param offset
+     * @param maxResults
+     * @param filter
+     * @param mimeTypes
+     * @param extensions
+     * @param showWorking
+     * @param showArchived
+     * @param noFolders
+     * @param onlyFiles
+     * @param sortBy
+     * @param sortByDesc
+     * @return
+     * @throws DotSecurityException
+     * @throws DotDataException
+     */
+    @Deprecated
+    public Map<String, Object> getFolderContent(final User user, final String folderId,
+                                                final int offset, final int maxResults, final String filter, final List<String> mimeTypes,
+                                                final List<String> extensions, final boolean showWorking, final boolean showArchived, final boolean noFolders,
+                                                final boolean onlyFiles, final String sortBy, final boolean sortByDesc)
+			throws DotSecurityException, DotDataException {
+
+    	return getFolderContent(user, folderId, offset, maxResults, filter, mimeTypes, extensions, showWorking, showArchived, noFolders, onlyFiles, sortBy, sortByDesc, false, 0);
+    }
+
+    /**
+     * @deprecated see {@link #getFolderContent(BrowserQuery)}
+     * @param user
+     * @param folderId
+     * @param offset
+     * @param maxResults
+     * @param filter
+     * @param mimeTypes
+     * @param extensions
+     * @param showWorking
+     * @param showArchived
+     * @param noFolders
+     * @param onlyFiles
+     * @param sortBy
+     * @param sortByDesc
+     * @param languageId
+     * @return
+     * @throws DotSecurityException
+     * @throws DotDataException
+     */
+    @Deprecated
+    public Map<String, Object> getFolderContent(final User user, final String folderId,
+                                                final int offset, final int maxResults, final String filter, final List<String> mimeTypes,
+                                                final List<String> extensions, final boolean showWorking, final boolean showArchived, final boolean noFolders,
+                                                final boolean onlyFiles, final String sortBy, final boolean sortByDesc, final long languageId)
+			throws DotSecurityException, DotDataException {
+
+    	return getFolderContent(user, folderId, offset, maxResults, filter, mimeTypes, extensions, showWorking, showArchived, noFolders, onlyFiles, sortBy, sortByDesc, false, languageId);
+    }
+    
+    protected class WfData {
+        
+        List<WorkflowAction> wfActions = new ArrayList<>();
+        boolean contentEditable = false;
+        List<Map<String, Object>> wfActionMapList = new ArrayList<>();
+        boolean skip=false;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid using redundant field initializer for 'skip'](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5337824)